model/shared/src/test/scala/explore/model/arb/ArbFocused.scala

```diff
@@ -30,6 +33,12 @@ trait ArbFocused {
   val focusedTargetGen: Gen[Focused.FocusedTarget] =
     arbitrary[Target.Id].map(FocusedTarget.apply)
 
+  val focusedAsterismGen: Gen[Focused.FocusedAsterism] =
+    arbitrary[Asterism.Id].map(FocusedAsterism.apply)
```

Comment:
Good catch. This was missing.

Reply:
We are still missing `focusedAsterismGen` and `focusedConstraintSetGen` in line 27. Could you add those please?

Reply:
Sure.